### PR TITLE
spec get index

### PR DESCRIPTION
**Pull request by frankburgs**

### CODE REVIEW

**Review by elysemontano — CHANGES_REQUESTED** [review]
Let's take a closer look at your request spec.  A few things to pay attention to is what is the url that you are pulling the apartments from?  Apartmentindex is the route on the frontend, but not where the data lives on the backend which is where you are making the request to.  Also, this test usually won't pass unless there is a user associated with the apartment.

File: spec/requests/apartments_spec.rb [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
 RSpec.describe "Apartments", type: :request do
+
+    describe "GET /index" do
+
+        it "gets a list of aparments" do
+
+          user = User.create(email: "test@example.com", password: "password", password_confirmation: "password")
+
+          user.apartments.create(
+            user_id: nil,
```
Comment (elysemontano) [inline spec/requests/apartments_spec.rb]: Since user.apartments will create the user, this line can get deleted.

**Review by elysemontano — CHANGES_REQUESTED** [review]
One more comment to address in apartments_spec

File: spec/requests/apartments_spec.rb (hunk 1 above)
Comment (frankburgs) [reply spec/requests/apartments_spec.rb]: If we delete line 9, what do we replace "user" with on line 11?

Comment (elysemontano) [reply spec/requests/apartments_spec.rb]: creating with the association on line 11 automatically creates a user id for that apartment.

**Review by elysemontano — CHANGES_REQUESTED** [review]

**Review by elysemontano — APPROVED** [review]

**Pull request merged**